test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -14,6 +14,7 @@ def models
   end
 
   def host_index_and_show(host)
+    Organization.any_instance.stubs(:simple_content_access?).returns(false)
```

Comment:
Are the stubs in this class necessary? You didn't change the API in this PR..

Reply:
@jturel If I take it out, I am getting a VCR failure shown below:
```
  1) Failure:
Api::V2::HostsControllerTest#test_with_content [/home/vagrant/katello/test/controllers/api/v2/hosts_controller_test.rb:19]:
Expected {"message"=>"\n\n================================================================================\nAn HTTP request has been made that VCR does not know how to handle:\n  GET https://localhost:8443/candlepin/owners/Empty_Organization\n\nThere is currently no cassette in use. There are a few ways\nyou can configure VCR to handle this request:\n\n  * If you're surprised VCR is raising this error\n    and want insight about how VCR attempted to handle the request,\n    you can use the debug_logger configuration option to log more details [1].\n  * If you want VCR to record this request and play it back during future test\n    runs, you should wrap your test (or this portion of your test) in a\n    `VCR.use_cassette` block [2].\n  * If you only want VCR to handle requests made while a cassette is in use,\n    configure `allow_http_connections_when_no_cassette = true`. VCR will\n    ignore this request since it is made when there is no cassette [3].\n  * If you want VCR to ignore this request (and others like it), you can\n    set an `ignore_request` callback [4].\n\n[1] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/debug-logging\n[2] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/getting-started\n[3] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/allow-http-connections-when-no-cassette\n[4] https://www.relishapp.com/vcr/vcr/v/3-0-3/docs/configuration/ignore-request\n================================================================================\n\n"} to be nil.
```

I did clean remove it in a few other areas that it was not needed.

Reply:
Can you look at the stack trace in test.log and see where the error is raised from? I'm missing how your changes would be affecting the hosts API at all